geopandas/sindex.py

```diff
@@ -168,41 +168,60 @@ def query_bulk(self, geometry, predicate=None, sort=False):
         """
         raise NotImplementedError
 
-    def intersection(self, coordinates):
-        """Compatibility wrapper for rtree.index.Index.intersection,
-        use ``query`` instead.
+    def nearest(self, geometry):
+        """
+        Returns the nearest geometry to the input geometries.
+
+        .. note::
+            ``nearest_all`` currently only works with PyGEOS >= 0.10.
+
+        Note that if pygeos is not available, geopandas
+        will use rtree for the spatial index, where nearest temporarily has a different
+        function signature to temporarily preserve existing functionality.
 
         Parameters
         ----------
-        coordinates : sequence or array
-            Sequence of the form (min_x, min_y, max_x, max_y)
-            to query a rectangle or (x, y) to query a point.
+        geometry : {shapely geometry, GeoSeries, GeometryArray, numpy.array of PyGEOS
+            geometries}
+            A single shapely geometry, one of the GeoPandas geometry iterables
```

Comment:
Side note, but can you update `nearest_all` docstring with the same explanation as this one here? (nearest_all also accepts a single geometry)